WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -774,7 +790,6 @@ nick_the_philosopher = {
 			trait = imbecile
 			trait = inbred
 			trait = incapable
-			religion = catholic
 			religion_group = pagan_group
```

Comment:
I did not say that Catholicism as a whole focused on philosophy. I said that a Catholic character who meets the requirements of the Nickname should BE ABLE to get it.